readme.md

```diff
@@ -482,21 +465,29 @@ class User
   end
 
   def User.new
-    puts "I've hijacked a class method!"
+    puts "I've hijacked a class method and broken core functionality!"
```

Comment:
might just leave this method out all together. might just stand to conflate doesn't really add value imo